frontend/public/components/image-stream.tsx

```diff
@@ -142,14 +142,23 @@ export const ExampleDockerCommandPopover: React.FC<ImageStreamManipulationHelpPr
   </Popover>;
 };
 
+const getImportErrors = (imageStream) => {
+  return _.transform(imageStream.status.tags, (acc, tag: any) => {
```

Comment:
Tiny nit: Any qualms with `result` instead of `acc`?  Or just `errors`.

Reply:
fwiw, we use `acc` with `reduce` a lot in console

Reply:
Ok! Works for me.  
I was initially trying to decide if it was an acronym for something :)

Reply:
https://lodash.com/docs/4.17.11#reduce

> The iteratee is invoked with four arguments:
(accumulator, value, index|key, collection).

Reply:
Yup!  Figured as much. I'm used to `result`, so my bad. No qualms with this.